src/core/include/openvino/op/scatter_elements_update.hpp

```diff
@@ -80,7 +87,12 @@ class OPENVINO_API ScatterElementsUpdate : public op::util::ScatterElementsUpdat
 
     bool has_evaluate() const override;
 
+    OPENVINO_SUPPRESS_DEPRECATED_START
+    bool evaluate(const HostTensorVector& outputs, const HostTensorVector& inputs) const override;
```

Comment:
This would break the API of the previous op, that's why I decided to go with the host tensors. I think the rewrite of all evaluate methods should be done at some point, probably along with the removal of the deprecated classes.